### ISSUE TITLE
Disabling items from “More tools & options” in Gutenberg blocks for non admin users

### ISSUE THREAD

**Issue by kanlukasz** [WordPress/gutenberg#26254]
Hello

( _I dug through the handbook, all forums, but still no effect_ )

I would like to disable items from "More tools & options" for non admin users. There are a lot of settings that I don't want to grant access to non-admin users

I mean this area:
![obraz](https://user-images.githubusercontent.com/30685349/96374787-3308e980-1175-11eb-8dd2-0bc4e7947852.png)

Is it possible?
Is there any filter or something?

**Comment by gziolo** [comment]
Yes, I don't think it's possible at the moment. Can you elaborate what would you like to control exactly? How do you see API working?

**Comment by kanlukasz** [comment]
From my point of view as a developer and as agency, it would be nice to be able to choose what we want to show to the end-client who can edit their website.

For example - we do not want to provide the ability to change `Options` or use `Manage all reusable blocks` or `Block Manager`.

Another thing is that we don't want to give a chance to change the `VIEW` option, because we want to decide how the edit panel will look to the end-client.

---
A good example is how we can disable the area and functionality of the `EDITOR`:
```
<?php

/**
 * @link https://developer.wordpress.org/reference/hooks/block_editor_settings/
 */
function test_custom_editor_settings($settings)
{
	$settings['codeEditingEnabled'] = false;
	return $settings;
}
add_filter('block_editor_settings', 'test_custom_editor_settings', 10, 2);

```
---

Non-technical users (i mean clients) should be kept to a minimum. Leaving something unnecessary for them only creates unnecessary overhead for explaining or saying "don't touch it"

I think an interesting option would be to be able to use a filter. Let there be an array from which we can exclude unwanted elements (this is a strong simplification, but I am only saying how I see it)

**Comment by gziolo** [comment]
@mtias, is it something that was discussed previously? We introduced a more fine-grained control for individual blocks through `theme.json` but the features listed aren't necessarily theme-specific but rather specific to a site or a user role.
